api.md

```diff
@@ -978,7 +978,7 @@ manually](#compress_chunk), instead of automatically as they age.
 
 #### Restrictions
 
-Version 1.5 does not support altering or inserting data into compressed
+Current version does not support altering or inserting data into compressed
```

Comment:
I think there should be an article here `The current version` but maybe have native english speaker chime in

Reply:
Good point. Missed it. Fixed.